### ISSUE TITLE
 Netatmo: NoClassDefFoundError javax.net and javax.net.ssl

### ISSUE THREAD

**Issue by IOOOTAlan** [openhab/openhab-addons#1070]
Greetings,

I'm currently testing the netatmo binding inside a custom karaf-4.0.4 environment and I kept having runtime errors where the binding would fail to initialize, reporting a cryptic `javax/net/ssl/HostNameVerifier` message next to the `Initializing` string.
I then fired up a debug session and I found out that a `NoClassDefFoundError` was thrown as soon as the `ApiClient` object was created. I then looked the `MANIFEST.MF` of the project and found that the `Import-Package` directive for `javax.net.ssl` was missing, so I added it.
After this, another class inside the `javax.net` package was not found, so I added it to the `MANIFEST.MF` too.
After these two additions, the binding started working correctly.
Is this a bug?
If needed, I can submit a pull request with these changes.

**Comment by kaikreuzer** [comment]
Sounds like a bug to me, so happy to accept a PR for it!

**Comment by doandzhi** [comment]
Greets,

I included the Netatmo binding in my custom distribution using the Eclipse Smarthome core and things worked out great with plugins taken from the openHAB-addons project. The thing that concerns me is that I reproduced this issue using the 2.2 version of the binding. I saw that in [this PR](https://github.com/openhab/openhab2-addons/pull/2682/files) the 2 packages (javax.net and javax.net.ssl) were removed and after adding them in the manifest the binding works properly. Were there a specific reason to remove them actually because it would be better to decouple the bindings from the openHAB core.
